src/els_parser.erl

```diff
@@ -56,7 +56,7 @@ parse_form(IoDevice, Location0, Parser, _Options) ->
           {ok, POIs, Location1}
       catch
         _:_ ->
-          {ok, [], Location1}
+          {ok, find_attribute_tokens(Tokens), Location1}
```

Comment:
This is to handle the case when there is a syntax error in the export list, because an entry is not yet written but we still need the information about the section as one of the POIs.

```
-export([ foo/1
        ,  b
        ]).
```
```